pkg/driver/mocks/mock_mount.go

```diff
@@ -88,63 +89,91 @@ func (m *MockMounter) List() ([]mount.MountPoint, error) {
 	return ret0, ret1
 }
 
-// List indicates an expected call of List
+// List indicates an expected call of List.
 func (mr *MockMounterMockRecorder) List() *gomock.Call {
 	mr.mock.ctrl.T.Helper()
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "List", reflect.TypeOf((*MockMounter)(nil).List))
 }
 
-// MakeDir mocks base method
+// MakeDir mocks base method.
 func (m *MockMounter) MakeDir(arg0 string) error {
 	m.ctrl.T.Helper()
 	ret := m.ctrl.Call(m, "MakeDir", arg0)
 	ret0, _ := ret[0].(error)
 	return ret0
 }
 
-// MakeDir indicates an expected call of MakeDir
+// MakeDir indicates an expected call of MakeDir.
 func (mr *MockMounterMockRecorder) MakeDir(arg0 interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "MakeDir", reflect.TypeOf((*MockMounter)(nil).MakeDir), arg0)
 }
 
-// Mount mocks base method
+// Mount mocks base method.
 func (m *MockMounter) Mount(arg0, arg1, arg2 string, arg3 []string) error {
 	m.ctrl.T.Helper()
 	ret := m.ctrl.Call(m, "Mount", arg0, arg1, arg2, arg3)
 	ret0, _ := ret[0].(error)
 	return ret0
 }
 
-// Mount indicates an expected call of Mount
+// Mount indicates an expected call of Mount.
 func (mr *MockMounterMockRecorder) Mount(arg0, arg1, arg2, arg3 interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Mount", reflect.TypeOf((*MockMounter)(nil).Mount), arg0, arg1, arg2, arg3)
 }
 
-// MountSensitive mocks base method
+// MountSensitive mocks base method.
 func (m *MockMounter) MountSensitive(arg0, arg1, arg2 string, arg3, arg4 []string) error {
 	m.ctrl.T.Helper()
 	ret := m.ctrl.Call(m, "MountSensitive", arg0, arg1, arg2, arg3, arg4)
 	ret0, _ := ret[0].(error)
 	return ret0
 }
 
-// MountSensitive indicates an expected call of MountSensitive
+// MountSensitive indicates an expected call of MountSensitive.
 func (mr *MockMounterMockRecorder) MountSensitive(arg0, arg1, arg2, arg3, arg4 interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "MountSensitive", reflect.TypeOf((*MockMounter)(nil).MountSensitive), arg0, arg1, arg2, arg3, arg4)
 }
 
-// Unmount mocks base method
+// MountSensitiveWithoutSystemd mocks base method.
+func (m *MockMounter) MountSensitiveWithoutSystemd(arg0, arg1, arg2 string, arg3, arg4 []string) error {
```

Comment:
Was this auto generated by mockgen?

Reply:
yes, I ran hack/update-gomock, else the tests failed